src/core/component.js

```diff
@@ -136,6 +126,38 @@ Component.prototype = {
     return utils.extend({}, data);
   },
 
+  updateCachedValue: function (value, isSinglePropSchema) {
+    var parsedValue;
+    if (value === '') {
+      this.attrValue = undefined;
```

Comment:
attr is the value the cached value you set with setAttribute. ```value``` is meaningless. All variables have a value